tools/python/maps_generator/README.md

```diff
@@ -12,22 +14,28 @@ Further we will call them world and coastlines.
 ## Setup
 You must have Python version not lower than 3.6 and complete the following steps:
 
-0. [Build and install generator_tool.](https://github.com/mapsme/omim/blob/master/docs/INSTALL.md#maps-generator)
-1. Change directory:
+0. Checkout to branch of your's app version (see note of #maps_generator section). 
```

Comment:
s/Checkout/Switch/
s/your's app/your app's/

Reply:
s/branch/the branch/

Reply:
s/note/the note/

Reply:
fixed